framework/src/Volo.Abp.Core/Volo/Abp/Localization/CultureHelper.cs

```diff
@@ -35,6 +36,10 @@ public static IDisposable Use([NotNull] CultureInfo culture, CultureInfo uiCultu
             });
         }
 
+        public static bool IsRtl
+        {
+            get { return Thread.CurrentThread.CurrentUICulture.TextInfo.IsRightToLeft; }
```

Comment:
Why not use `CultureInfo.CurrentUICulture`?

Reply:
Yes, we can but need to be tested.

Actually, I took the code from aspnetzero as it is :)

Reply:
Because `CultureHelper`  is using `CultureInfo` now. : )

Reply:
Ok, I will change it and push it to merge.

Reply:
Hi @hikalkan @maliming, V2.7 MVC template crashes after upgrade from V2.6

The type or namespace name 'Localization' does not exist in the namespace 'Volo.Abp.AspNetCore' (are you missing an assembly reference?)
+
@using Volo.Abp.AspNetCore.Localization
The name 'CultureHelper' does not exist in the current context
+
<html lang="@CultureInfo.CurrentCulture.Name" dir=@(CultureHelper.IsRtl ? "rtl" : "")>

Also, I got better bootstrap RTL library [https://www.npmjs.com/package/bootstrap-v4-rtl](url)
I made a new style bundle contributor that replaces the bootstrap.css if RTL language.
My question where can I place it at the framework or leave it at the template?

Can I update the pull request if agreed with the new changes?

Thanks,
Khaled

Reply:
`CultureHelper` exists in the` Volo.Abp.Core` package, its namespace is `Volo.Abp.Localization`.

Reply:
`style bundle contributor` should be in the framework.

eg: `jstree package & bundle contributor. `https://github.com/abpframework/abp/commit/72b9b43e9130c098a085dbe563a231e8eb6e2f92